.github/workflows/fcos-podman-next-build.yml

```diff
@@ -0,0 +1,52 @@
+name: Build FCOS image with packages from rhcontainerbot/podman-next
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  IMAGE_NAME: fcos
+  IMAGE_TAGS: latest next podman-next ${{ github.sha }}
```

Comment:
Minor: I don't really see a need for `next` or `podman-next`, since `latest` will be effectively the exact same thing.

Also Minor: Using a "naked" commit sha as a tag makes me slightly uncomfortable.  It makes sense as an annotation or label for sure.  But any lay-person looking at an image named "fcos" on quay, will likely have no clue what the sha belongs to.   I'm not 100% on what the intended use of this image is, but I would think something like the full podman version would be more helpful.

Since you're setting an expiration of these tags, it's fine to fixup this stuff later.

Reply:
Hmm, now that  I think of it, someone browsing: `quay.io/podman/fcos` and looking at the tag `latest` would assume it's stable, like for example: `fedora:latest` points to fedora's latest release. Maybe latest should be removed. If I gotta choose between next and podman-next, I'll choose podman-next.

Reply:
Ya, `podman-next' is way more meaningful than 'latest'